packages/dropdown-react/example/index.tsx

```diff
@@ -18,28 +18,47 @@ const DropdownDemo = () => {
         <>
             <Dropdown
                 inline
+                className="jkl-spacing--all-2"
                 label="The Room scene"
                 defaultPrompt="Choose your favorite"
                 items={items}
                 onChange={setFavoriteScene}
                 helpLabel="The room is the greatest movie"
                 errorLabel={favoriteScene !== "" ? "You can't pick, they are all the best" : undefined}
+                variant="secondary"
+            />
+            <Dropdown
+                inline
+                className="jkl-spacing--all-2"
+                label="Fødselsår"
+                items={years}
+                initialInputValue="1986"
+                variant="small"
             />
-            <Dropdown inline label="Fødselsår" items={years} initialInputValue="1986" />
 
-            <Select label="Standard select" items={items} onChange={(e) => console.log(e.target.value)} />
             <Select
+                className="jkl-spacing--top-5"
+                label="Native select"
+                items={items}
+                onChange={(e) => console.log(e.target.value)}
+                value="ChirpChirpChirp"
+            />
+            <Select
+                className="jkl-spacing--top-3"
                 helpLabel="The room is the greatest movie"
                 errorLabel={favoriteScene !== "" ? "You can't pick, they are all the best" : undefined}
-                label="Standard select"
+                label="Native select"
                 items={items}
-                onChange={(e) => console.log(e.target.value)}
+                onChange={(e) => setFavoriteScene(e.target.value)}
+                placeholder="Choose your favorite"
+                variant="secondary"
+                value={favoriteScene}
             />
 
-            <Dropdown label="Favorite The Room scene" items={items} />
-            <Dropdown className="short" label="Fødselsår" items={years} />
+            <Dropdown className="jkl-spacing--top-5" label="Favorite The Room scene" items={items} />
+            <Dropdown className="short jkl-spacing--top-3" label="Fødselsår" items={years} />
```

Comment:
short?

Reply:
I made a support class for the example (long time ago) to show that width can be adjusted by custom `className`s. I agree that it might be a bit confusing…